landing-page/landing-page.js

```diff
@@ -0,0 +1,16 @@
+let rightArrow = document.querySelector(`.right-arrow`);
+let leftArrow = document.querySelector(`.left-arrow`);
+
+console.log(rightArrow);
+
+rightArrow.addEventListener(`click`, goToMusicPage);
+
+leftArrow.addEventListener(`click`, goToGamePage);
+
+function goToMusicPage () {
+  window.open(`./home-music/home-music.html`, `_self`);
+}
+
+function goToGamePage () {
+  window.open(`./home-game/home-game.html`, `_self`);
+}
```

Comment:
You should also make the images clickable and lead to the game home and music home page on the tablets screens and the screens, or make the arrows visible.